### ISSUE TITLE
Resolve unhackable story series & event series urls

### ISSUE THREAD

**Issue by LaurenFBaily** [wellcomecollection/wellcomecollection.org#11214]
### What
Decide on solution for resolving the non-hackable urls for story series and event series. Current options:

1. Replace the /series/ part of the url with /stories/ and /event-series/ with /events/ 
2. Create index pages for both

### Why
Links aren't hackable so you get a 404 if you try and access www.wellcomecollection.org/event-series or www.wellcomecollection.org/series which is not best practice.

### Background
[See epic on Notion](https://www.notion.so/wellcometrust/Descriptive-urls-ed7cf0014a584b9ab7a9f9b8e96d3742?pvs=4)

**Comment by rcantin-w** [comment]
Is this dependant on https://github.com/wellcomecollection/wellcomecollection.org/issues/11206 ? Just remember saying we couldn't really do that as series and articles could have the same uid and that the easiest was to have a listing page for series (`/series/stories` being the url for stories though)

**Comment by LaurenFBaily** [comment]
Yes, I included that as the second option above. I've put this in the Icebox for now as I think it's the least valuable change.

**Comment by rcantin-w** [comment]
Ah of course 🤦‍♀️ pre-coffee read!
